index.html

```diff
@@ -54,7 +55,189 @@ <h2 class="proffession">I am a Software Developer</h2>
             </li>
           </ul>
         </section>
-      </main>
     </div>
+
+
+<section class="works"> 
+    <div class="grid-container">
+      
+        
+          <div class="item0">
+            <h2 class="recents">My Recent Works</h2>
+            <img src="images/Img Placeholder.png" alt="placeholder">
+          </div>
+        
+
+        
+          <div class=" item1">
+            <h2>Multi-Post Stories</h2>
+            <p class="stories">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                  <div class="lang">
+            <ul>
+              <li><button>css</button></li>
+              <li><button>html</button></li>
+              <li><button>bootstrap</button></li>
+              <li><button>Ruby</button></li>
+            </ul>
+                  </div>
+                  <button class="see-project">See Project</button>
+          </div>
+        
+
+      <div class="grid item2">
+        <h2>Project Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="lang">
+          <ul>
+            <li><button>html</button></li>
+            <li><button>bootstrap</button></li>
+            <li><button>Ruby</button></li>
+          </ul>
+        </div>
+        
+        <button class="see-project btn">See Project</button>
+      </div>
+     
+
+      <div class="grid item3">
+        <h2>Project Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="lang">
+          <ul>
+            <li><button>html</button></li>
+            <li><button>bootstrap</button></li>
+            <li><button>Ruby</button></li>
+          </ul>
+        </div>
+        <button class="see-project btn">See Project</button>
+      </div>
+      
+
+      <div class="grid item4">
+        <h2>Project Art Printing Data</h2>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="lang">
+          <ul>
+            <li><button>html</button></li>
+            <li><button>bootstrap</button></li>
+            <li><button>Ruby</button></li>
+          </ul>
+        </div>
+        <button class="see-project btn">See Project</button>
+      </div>
```

Comment:
- You have done well aligning the content of your project cards using the `flex` display property, but there are some few differences you still need to address. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your Project 👇:
- The background color for the list of technologies in your project is different from what is in the design template, please fix this for all project cards 👍 👍 
- I suggest you consider setting the borders for your project buttons to `none` so they can look closer to what is in the design template 🙏 
<img width="277" alt="rasraet" src="https://user-images.githubusercontent.com/73607512/205457492-62b316cc-ea8d-484f-b852-b9209604e802.PNG">

Figma Design:
<img width="186" alt="utuktu" src="https://user-images.githubusercontent.com/73607512/205457578-5e3efb3c-0638-4393-b769-df8f6b093d98.PNG">

Reply:
- [ ] Please you still need to work on this list as earlier suggested by the previous reviewer, kindly add a padding to the parent container. 👍